demo/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>EME Encryption Scheme Polyfill Demo</title>
+
+    <!-- Load & install the polyfill. -->
+    <script src="../dist/eme-encryption-scheme-polyfill.js"></script>
+    <script>EmeEncryptionSchemePolyfill.install();</script>
+
+    <!-- Load web components used in the demo. -->
+    <script type="module" src="https://unpkg.com/elix@^9/define/AutoCompleteComboBox.js"></script>
+
+    <!-- Load the code for the query part of the demo. -->
+    <script src="query.js"></script>
+
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>EME Encryption Scheme Polyfill Demo</h1>
+
+    <h2>Test query</h2>
+    <h3>Choose parameters like key system, encryption scheme, and media types,
```

Comment:
This seems like it should be a `<div>` or `<p>` and not a header.  Headers are usually used for short titles of text.

Reply:
You make a good point.  Done.